### PR TITLE
Add Faker.Vehicle

### PR DESCRIPTION
**Pull request by daytonn**
Added the `Faker.Vehicle` module which adds several functions to generate realistic vehicle data.

### CODE REVIEW

File: lib/faker/vehicle.ex [hunk 1]
```diff
@@ -0,0 +1,224 @@
...
+
+  @doc """
+  Returns a vehicle drivetrain string
+
+  ## Examples
+      iex> Faker.Vehicle.drivetrain()
+      "4x2/2-wheel drive"
+      iex> Faker.Vehicle.drivetrain()
+      "4x4/4-wheel drive"
+  """
```
Comment (vbrazo) [inline lib/faker/vehicle.ex]: As a standard rule, we work with 4 examples. Please double check your examples.

Comment (daytonn) [reply lib/faker/vehicle.ex]: I added some more examples :+1:

**Comment by daytonn** [comment]
@vbrazo Is there anything else I need to do to get this pulled? Is this project still actively maintained?

**Comment by anthonator** [comment]
Yes, this project is actively maintained by _volunteers_.

You currently have merge conflicts on `CHANGELOG.md`. Would you mind fixing those? I'm sure @vbrazo would be happy to review this again and give a 👍 👎 when he has a free minute.

**Comment by daytonn** [comment]
I do appreciate that this is maintained by volunteers. I was just wondering because there doesn't seem like there's been a lot of activity lately/

**Comment by anthonator** [comment]
I don't know much about cars so I can't really comment on that part (although I am glad the Nissan Altima is represented) but the code looks good to me.

@vbrazo let us know if you have any additional comments.

**Comment by vbrazo** [comment]
It'd be nice if we could have a method for `license plate`.

**Pull request merged**